lib/active_merchant/billing/gateways/paypal_express_common.rb

```diff
@@ -16,13 +16,11 @@ def redirect_url
         test? ? test_redirect_url : live_redirect_url
       end
 
-      def redirect_url_for(token, options = {})
+      def base_redirect_url_for(token, options: {}, cmd: nil)
         options = {:review => true, :mobile => false}.update(options)
```

Comment:
What I am saying is that `:mobile => false` doesn't make sense here because it isn't even used :arrow_down: . It was bumped up to the two other functions and now pass in `cmd`.